addons/account/tests/test_chart_template.py

```diff
@@ -692,7 +692,7 @@ def test_branch(self):
         }])
 
         with patch.object(AccountChartTemplate, '_get_chart_template_data', side_effect=test_get_data, autospec=True):
-            self.env['account.chart.template'].try_loading('test', company=company, install_demo=True)
+            self.env['account.chart.template'].try_loading('test', company=company, install_demo=False)
```

Comment:
We want this to make sure that we can do all standard operations on branches.

I'd prefer to have `_get_demo_data` working when no demo data is loaded in the dependencies of `account` somehow. (we mainly need products and partners)

Otherwise, we could use another set of data completely independent of demo and used only in tests...

Reply:
I am not sure why `base.res_partner_3` specifically is being used, and this is too specific to account module for me, could you take this branch over?